db/schema.rb

```diff
@@ -11,7 +11,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20161104013922) do
+ActiveRecord::Schema.define(version: 20161110011213) do
```

Comment:
Apparently this version number was causing the merge conflict, but I wasn't sure what to do about it. I thought if I tried to merge into my local master, it would fail because of this, but... it didn't :=} So now I've got a master branch on my working repo that has UA-135 merged into it.